sdk/include/psdk/shellapi.h

```diff
@@ -281,12 +279,19 @@ typedef struct _NOTIFYICONDATAW {
 #endif
 } NOTIFYICONDATAW,*PNOTIFYICONDATAW;
 
+/* NOTIFYICONDATA sizes to force older than current structs */
```

Comment:
This is not forcing anything, it can be used to check which version of the struct is provided by the application. (And thus, which fields can be used)